.github/workflows/maven-deploy.yml

```diff
@@ -21,15 +21,13 @@ on:
         description: internal or external repository
   push:
     branches:
-     - 'develop'
-     - 'test/ci'
-
-
+      - 'develop'
+      - 'test/ci'
 
 jobs:
-  maven-deploy-internal:
+  maven-deploy-push:
```

Comment:
The name `push` confuses me.

Reply:
I agree, it is confusing because we also use `push` for the push server component in our naming.

Reply:
I will gladly rename it - this is a naming used from crypto (https://github.com/wultra/powerauth-crypto/pull/439/files)